src/libslic3r/PrintConfig.cpp

```diff
@@ -2810,6 +2810,56 @@ void PrintConfigDef::init_fff_params()
     def->mode     = comAdvanced;
     def->set_default_value(new ConfigOptionFloat(0.));
 
+    def           = this->add("interlocking_beam", coBool);
+    def->label    = L("Use beam interlocking");
```

Comment:
would it be better to use "Generate interlocking structure" here? Same as Cura.

Reply:
Because there is already some type of interlocking in Orca, by generating overlapping layers. hens this option as a different type of interlocking.

Reply:
I see